### PR TITLE
Convert data migration to Django migration

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@macdiesel @doctoryes @symbolist @muzaffaryousaf look ok?

**Comment by doctoryes** [comment]
:+1:

**Comment by muzaffaryousaf** [comment]
@nedbat We already have this data migration https://github.com/edx/edx-milestones/pull/13/files#diff-120ce99cd17baa9e8790a252b7ea9436R13 .

**Comment by nedbat** [comment]
@muzaffaryousaf Thanks, I had overlooked that.  I guess I'd be more comfortable keeping the data migration separate from the schema migration.

**Comment by muzaffaryousaf** [comment]
@nedbat Yes, that totally make sense but we have to remove the first one.

File: milestones/migrations/0002_data__seed_relationship_types.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        )
+
+
+def delete_relationship_types(apps, schema_editor):
+    """
+    A backwards migration of seeded MilestoneRelationshipType data is difficult due to the
+    likelihood of foreign key dependencies established over time.  The good news is the next
+    database state we'll enter is initial (ie, no further schema changes), so the data can stay.
+    """
+    pass
```
Comment (muzaffaryousaf) [inline milestones/migrations/0002_data__seed_relationship_types.py]: Why we deleted the reverse part ?

**Comment by nedbat** [comment]
@muzaffaryousaf updated, look ok?

**Pull request merged**